drop-in/src/main/java/com/adyen/checkout/dropin/ui/DropInViewModel.kt

```diff
@@ -74,7 +88,7 @@ class DropInViewModel(private val savedStateHandle: SavedStateHandle) : ViewMode
             savedStateHandle[IS_WAITING_FOR_RESULT_KEY] = value
         }
 
-    var cachedGiftCardComponentState: GiftCardComponentState?
+    private var cachedGiftCardComponentState: GiftCardComponentState?
```

Comment:
we never reset this. is there possibly a case where we need to reset this to `null`?

Reply:
We don't need to reset now it but I don't like how it is now so I'm refactoring this part in the next PR